api.go

```diff
@@ -238,6 +241,9 @@ type Repository struct {
 	// Importance of the repository, bigger is more important
 	Rank uint16
 
+	// SOURCEGRAPH Priority of the repository, bigger is more important.
+	Priority float64
+
```

Comment:
Would it be worth reusing `Rank` here? We don't really need a float64 if we encode like upstream/zoekt. WDYT?

Reply:
Also curious about @stefanhengl's point here. What prevents us from reusing Rank? On a side note, I also prefer that terminology compared to priority.